fetch_raw_data.py

```diff
@@ -216,6 +216,11 @@ def fetch_from_facebook(user, google_cloud_credentials_file_path, raw_data_dir,
                     facebook.get_all_comments_on_post(post_id, raw_export_log_file=raw_comments_log_file)
                 )
 
+            # Download the post and add it as context to all the comments
```

Comment:
what's the purpose of this?

Reply:
It's so we can see what each comment was in response to in the pipeline. In the future, this would give us easy access to the post text so we can easily show the context of the comment. For now it lets us derive the post type (image vs. video) which the comment is in response to. I'll update the comment to make this more clear.

Reply:
Okay thanks

Reply:
Added